recipes/dataconf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dataconf" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dataconf-{{ version }}.tar.gz
+  sha256: c4f845b405a4b5ddbf59267879dd5a9e1a1b4ee73fff9c0e0df5ca560f8b2705
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - dataconf = dataconf.cli:run
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - pyhocon >=0.3.54,<0.4.0
+    - python >=3.6,<4.0
```

Comment:
Upstream says `>=3.8`?

Reply:
I'll change this, I must have missed checking this and just used default from GS